README.md

```diff
@@ -49,32 +55,49 @@ httpserver 8080 #eg with: alias httpserver='python -m SimpleHTTPServer'
 open http://localhost:8080
 ```
 
-## 2. Work with local compile server
+### 2. Local compile server setup
 
 ```
 git clone git@github.com:purescript/trypurescript.git
 cd trypurescript
 
 stack build
 
-# use one of the backends
-cd staging/core
-# get the sources of the deps
+# Install PureScript dependencies
+cd staging
 psc-package install
 
 # note: globs like **/src/** do not work
-stack exec trypurescript 8081 ".psc-package/psc-0.13.6/*/*/src/**/*.purs" "src/*.purs"
+stack exec trypurescript 8081 ".psc-package/psc-0.13.6-*/*/*/src/**/*.purs" "src/*.purs"
 # should output that is is compiling the sources (first time)
 # then: Setting phasers to stun... (port 8081) (ctrl-c to quit)
 ```
 
-## 3. Point client to local compile server
+## Server API
 
-(instead of the ones at try.purescript.org)
-```
-# edit API.purs
+The server is a very basic web service which wraps the PureScript compiler, allowing clients to send PureScript code to be compiled and receiving either compiled JS or error messages in response.
 
-  , compile: compile "http://localhost:8081"
-  , getBundle: getDefaultBundle "http://localhost:8081"
+### Compile PureScript code
 
-```
+**POST /compile**
+
+- Request body: PureScript code
+- Response body: Either `{ js: "..." }` or `{ error: "..." }`
+- Status code: 200 (success), even if compilation failed
+
+The response does not use error codes, to make it easier to use the API from another domain using CORS.
```

Comment:
I think in the context of a compilation service a "compiler error" would still be a success either way. So making it easier to do CORS is nice, but I wouldn't need that motivation here.